### PR TITLE
feat: allow ubuntu image creation on gcp

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What problem does this PR solve?**:

**Which issue(s) does this PR fix?**:

* https://jira.d2iq.com/browse/D2IQ-90150

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by supershal** [comment]
can you please add makefile targets for ubuntu in `images.mk` and `ci.mk`.
Once done, I can clone TC configuration to create job for ubuntu distros.

**Comment by supershal** [comment]
can you please add makefile targets for ubuntu in `images.mk` and `ci.mk`.
Once done, I can clone TC configuration to create job for ubuntu distros.

**Review by faiq — DISMISSED** [review]
Code looks fine but a CI target would be appreciated

File: make/images.mk [hunk 1]
```diff
@@ -461,6 +461,11 @@ ubuntu18:
 ubuntu18-azure:
```
Comment (faiq) [inline make/images.mk:461]: I think you're going to need to change this to `build-gcp-ubuntu-1804` 

What I did previously is map this one to the new target see here https://github.com/mesosphere/konvoy-image-builder/commit/bdbd8592e7f43b77c32960c95c607d12247d8f62#diff-1d0eb908228c37e62c17adb04039870d318703e16c43c002a9e6de2c981e5774

Comment (alejandroEsc) [reply make/images.mk]: great catch @faiq

**Review by supershal — APPROVED** [review]

File: make/images.mk [hunk 2]
```diff
@@ -457,9 +457,14 @@ ubuntu18:
 	$(MAKE) build-aws-ubuntu-18
 
 # Ubuntu 18(04) Azure
```
Comment (faiq) [inline make/images.mk:459]:
```suggestion
# Ubuntu 18(04) Azure
# preserve old make targets for CI 
.PHONY ubuntu18-azure
ubuntu18-azure: ubuntu1804-azure

.PHONY ubuntu20-azure
ubuntu20-azure: ubuntu2004-azure
```

File: make/images.mk [hunk 3]
```diff
@@ -457,9 +457,20 @@ ubuntu18:
...
-.PHONY: ubuntu18-azure
-ubuntu18-azure:
-	$(MAKE) build-azure-ubuntu-18
+# preserve old make targets for CI 
+.PHONY ubuntu18-azure
```
Comment (faiq) [inline make/images.mk:461]:
```suggestion
.PHONY: ubuntu18-azure
```

File: make/images.mk [hunk 4]
```diff
@@ -457,9 +457,20 @@ ubuntu18:
...
+ubuntu18-azure: ubuntu1804-azure
+
+.PHONY ubuntu20-azure
```
Comment (faiq) [inline make/images.mk:464]:
```suggestion
.PHONY: ubuntu20-azure
```

**Pull request merged**